### PR TITLE
Implement UI of landing page 

### LINKED ISSUE #80: FE: Landing Page
Make a landing page for potential sitters to opt in to pet sitting.

### PR DESCRIPTION
**Pull request by hayderjebur**
### What this PR does (required):
- UI of the landing page.
- Validate Use input.
- Made a reusable input component. 

### Screenshots / Videos (front-end only):
- Post at least one screenshot to show the feature in action. **Only required in front-end work**
Screen Shot 2021-10-27 at 1 14 52 PM
Scre
en Shot 2021-10-27 at 1 12 07 PM
### Any information needed to

 test this feature (required):
- Post what steps to take to be able to test this feature (eg. "sign up as a new user and upload a profile picture from the navbar")

### Any issues with the current functionality (optional):
- Used random an image for now

### CODE REVIEW

File: client/src/pages/landingPage/LandingPage.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+export default function LandingPage(): JSX.Element {
+  const today = new Date();
+  today.setHours(0, 0, 0, 0);
+  const classes = useStyles();
+
+  const handleSubmit = (
+    { where, dropIn, dropOff }: { where: string; dropIn: string; dropOff: string },
+    { setSubmitting }: FormikHelpers<{ where: string; dropIn: string; dropOff: string }>,
+  ): void => {
+    //TODO here we have all user inputs values(where,dropIn,dropOff) and ready to make a req to server
```
Comment (sundayezeilo) [inline client/src/pages/landingPage/LandingPage.tsx:21]: Let's remove comment

**Review by sundayezeilo — COMMENTED** [review]
Your landing page looks good! :+1:

File: client/src/context/useAuthContext.tsx [hunk 2]
```diff
@@ -50,7 +50,7 @@ export const AuthProvider: FunctionComponent = ({ children }): JSX.Element => {
         } else {
           // don't need to provide error feedback as this just means user doesn't have saved cookies or the cookies have not been authenticated on the backend
           setLoggedInUser(null);
-          history.push('/login');
+          history.push('/land');
```
Comment (rajivtitus) [inline client/src/context/useAuthContext.tsx:53]:
```suggestion
          history.push('/landing-page');
```
Is this a typo or do we have another page for the '/land' route? 😄

File: client/src/pages/landingPage/LandingPage.tsx [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    setSubmitting(false);
+  };
+
+  return (
+    <Grid container component="main" className={classes.root}>
+      <CssBaseline />
```
Comment (rajivtitus) [inline client/src/pages/landingPage/LandingPage.tsx:27]: Please remove the `CssBaseline` component as it is a global reset and needs to be included only once, ideally in the App.tsx

File: client/src/pages/landingPage/LandingPage.tsx [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+      <Grid item xs={12} sm={8} md={6} component={Paper} elevation={0}>
+        <Box className={classes.paper}>
+          <Typography className={classes.fontBold} variant="h2">
+            Find the care your dog deserves
+          </Typography>
+          <Formik
+            initialValues={{ where: '', dropIn: '', dropOff: '' }}
+            validationSchema={Yup.object().shape({
+              where: Yup.string().required('Where is required'),
+              dropIn: Yup.date().min(today, 'Drop in cannot be in the past').required(''),
```
Comment (rajivtitus) [inline client/src/pages/landingPage/LandingPage.tsx:38]: Nice work adding validation here 💯

File: client/src/pages/landingPage/LandingPage.tsx [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+                      error={touched.dropOff && Boolean(errors.dropOff)}
+                      variant="outlined"
+                      value={values.dropOff}
+                      handleChange={handleChange}
+                    />
+                  </Grid>
+                </Grid>
+
+                <Button type="submit" variant="contained" color="secondary" className={classes.submit}>
+                  FIND MY DOG SITTER
```
Comment (rajivtitus) [inline client/src/pages/landingPage/LandingPage.tsx:93]: Please avoid using CAPITALCASE text, you can use a CSS class to transform text instead

File: client/src/themes/theme.ts [hunk 6]
```diff
@@ -11,6 +11,7 @@ export const theme = createMuiTheme({
   },
   palette: {
     primary: { main: '#3A8DFF' },
+    secondary: { main: '#F54242' },
```
Comment (rajivtitus) [inline client/src/themes/theme.ts:14]: 👍🏽

**Review by rajivtitus — APPROVED** [review]
UI looks great! 🚀 
Please address a couple of minor comments prior to merging. Thank you!

File: client/src/context/useAuthContext.tsx (hunk 2 above)
Comment (hayderjebur) [reply client/src/context/useAuthContext.tsx:53]: I was testing something😬

**Pull request merged**